tests/config/config0.yml

```diff
@@ -0,0 +1,20 @@
+---
+input:
+  dataset_src: local
+  datasets:
+    dataset_0:
```

Comment:
I'm still not in favour of dataset_0 as the identifier but let's keep moving for now.